code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -416,6 +416,7 @@ GLOBAL_LIST_INIT(silk_recipes, list ( \
 	merge_type = /obj/item/stack/sheet/cloth/durathread
 	drop_sound = 'sound/items/handling/cloth_drop.ogg'
 	pickup_sound =  'sound/items/handling/cloth_pickup.ogg'
+	merge_type = /obj/item/stack/sheet/cloth/durathread
```

Comment:
?

Reply:
Merge type allows for it to merge its types and subtypes

Reply:
there's two of those

Reply:
> there's two of those

Whoops ill go fix it properly then in a bit